core/Project.toml

```diff
@@ -106,15 +118,5 @@ TestItemRunner = "f8b46487-2199-4994-9208-9a1283c18c0a"
 [preferences.LinearSolve]
 LoadMKL_JLL = false
 
-[preferences.OrdinaryDiffEq]
-PrecompileAutoSpecialize = false
-PrecompileAutoSwitch = false
-PrecompileDefaultSpecialize = false
-PrecompileFunctionWrapperSpecialize = false
-PrecompileLowStorage = false
-PrecompileNoSpecialize = false
-PrecompileNonStiff = false
-PrecompileStiff = false
-
```

Comment:
We'll precompile less since we load less solver packages. And for the packages we do load, maybe it's not so bad to precompile a bit so Ribasim starts faster in interactive sessions.